imports/both/api/MappingEvents/buildInvitationEmailFromTemplate.ts

```diff
@@ -0,0 +1,104 @@
+import {DraftEmailTemplateData, TemplateTypeEnum} from './DraftEmailTemplateData';
+import {generateAppLinkToEventUnchecked} from './generateAppLinkToEvent';
+import {t} from 'ttag';
+import IMappingEvent from './IMappingEvent';
+import IApp from '../Apps/IApp';
+import IOrganization from '../Organizations/IOrganization';
+import {Meteor} from 'meteor/meteor';
+import IMappingEventParticipant from '../MappingEventParticipants/IMappingEventParticipant';
+import {getDisplayedNameForUser} from '../../lib/user-name';
+
+type BodyTemplateData = {
+  appId: string,
+  appName: string,
+  eventId: string,
+  eventName: string,
+  organizationId: string,
+  organizationName: string,
+  token?: string
+  inviterUserName: string,
```

Comment:
Not sure, but maybe:

```suggestion
  hostUserName: string,
```

Reply:
Not necessarily true. The person that sends out the invite is might not be the host. But I wasn't happy with the name either.

Reply:
I've looked other synonyms up and didn't find anything better, so let's keep it as at least it's 100% clear what it means :)